case_studies/multiband/multiband_exps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Runs the experiments for the BLISS multiband paper.
+
+# train single-band model
+bliss -cp ~/bliss/case_studies/multiband -cn config mode=train \
+    train.trainer.logger.name=MULTIBAND \
+    train.trainer.logger.version=r_band \
+    encoder.image_normalizer.bands=[2] \
+    cached_simulator.cached_data_path=/data/scratch/regier/2percent
```

Comment:
Would you make `cached_data_path` a command line argument to make that is set to `/data/scratch/regier/2percent` by default, but which can be modified?